src/screens/NewTopicScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { postNewTopic}from '../api/DataHandler';
+
+//import { Link } from 'react-router-dom'
+
+interface NewTopicState{
+    post_title: string,
+    post_text: string,
+    post_url: string,
+    username: string,
+    pwdHash: string,
+    hanesst_id: number,
+}
+class NewTopicScreen extends React.Component<any, NewTopicState> {
+
+state = {
+    post_title: "",
+    post_text: "",
+    post_url: "",
+    username: "",
+    pwdHash: "",
+    hanesst_id: 0
+}
+
+    
+handlePostTitleInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({post_title: event.currentTarget.value})
+}
+
+handlePostTextInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({post_text: event.currentTarget.value})
+}
+
+handlePostUrlInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({post_url: event.currentTarget.value})
+}
+
+handleUserNameInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({username: event.currentTarget.value})
+}
+
+handlePWDInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({pwdHash: event.currentTarget.value})
+}
+
+handleHasesstIdInput(event:React.FormEvent<HTMLInputElement>){
+    this.setState({hanesst_id: event.currentTarget.valueAsNumber})
+}
+
+
+  
+handleNewTopicSubmit = async ( e: React.FormEvent<HTMLFormElement>):Promise<void> => {
+        e.preventDefault();
+        if(this.state.hanesst_id <= 0  ){
```

Comment:
We don't need hanesst_id for the frontend

Reply:
Function has been removed